tests/layer_tests/common/layer_test_class.py

```diff
@@ -24,14 +24,15 @@ def produce_model_path(self, framework_model, save_path):
     def get_framework_results(self, inputs_dict, model_path):
         pass
 
-    def _test(self, framework_model, ref_net, ie_device, precision, ir_version, temp_dir, infer_timeout=60,
-              enabled_transforms='', disabled_transforms='', **kwargs):
+    def _test(self, framework_model, ref_net, ie_device, precision, ir_version, temp_dir, use_new_frontend=False,
+              infer_timeout=60, enabled_transforms='', disabled_transforms='', **kwargs):
         """
-        :param enabled_transforms/disabled_transforms: string with idxs of transforms that should be enabled/disabled.
+        :param enabled_transforms/disabled_transforms: string with idxs of tra'nsforms that should be enabled/disabled.
                                                        Example: "transform_1,transform_2"
         """
         model_path = self.produce_model_path(framework_model=framework_model, save_path=temp_dir)
 
+        self.use_new_frontend = use_new_frontend
         # TODO Pass environment variables via subprocess environment
         os.environ['MO_ENABLED_TRANSFORMS'] = enabled_transforms
         os.environ['MO_DISABLED_TRANSFORMS'] = disabled_transforms
```

Comment:
please insert TODO or commented code to pass `use_new_frontend` into the MO tool

Reply:
the flag was passed to MO